doc/components/dropdown.md

```diff
@@ -26,6 +26,11 @@ like `text="Click me" icon="icon-filter"`. As following you can see an example o
   </div>
 </div>
 
+**Disabled**
+As any common html element the dropdown can be disabled by adding the `disabled="1"` flag to it.
```

Comment:
is this a React thing?
if it is about implementation do the check `props.disabled!==undefined` (or sth like that) this results in the same behaviour as native